config/spapi.php

```diff
@@ -1,8 +1,13 @@
 <?php
 
 return [
+    'registration_enabled' => env('SPAPI_REGISTRATION_ENABLED', true),
+
     'installation_type' => 'single',
 
+    'debug' => env('SPAPI_DEBUG', false),
+    'debug_file' => env('SPAPI_DEBUG_FILE', 'php://output'),
```

Comment:
Could you move these new config options to the end of the config file? I'm guessing fewer people will be interacting with them than with the rest of the config options. Also, a comment describing the purpose of the `registration_enabled` option would be great :)